tests/integration/hive_metastore/test_migrate.py

```diff
@@ -101,3 +102,46 @@ def test_migrate_external_table(ws, sql_backend, inventory_schema, make_catalog,
 
     target_tables = list(sql_backend.fetch(f"SHOW TABLES IN {dst_schema.full_name}"))
     assert len(target_tables) == 1
+
+
+@retried(on=[NotFound], timeout=timedelta(minutes=5))
+def test_revert_migrated_table(ws, sql_backend, inventory_schema, make_schema, make_table):
+    schema1 = make_schema(catalog_name="hive_metastore")
+    schema2 = make_schema(catalog_name="hive_metastore")
+    tables = [
+        make_table(schema_name=schema1.name),
+        make_table(schema_name=schema1.name),
+        make_table(schema_name=schema2.name),
+        make_table(schema_name=schema2.name),
+    ]
+
+    for i in range(3):
```

Comment:
For table in tables:

Reply:
I just want to cover the first 3.

Reply:
I changed it to slicing the original list